WordPress/src/main/AndroidManifest.xml

```diff
@@ -1148,11 +1148,6 @@
                 tools:node="remove" />
         </provider>
 
-        <!-- Scanner -->
-        <meta-data
-            android:name="com.google.mlkit.vision.DEPENDENCIES"
```

Comment:
I removed this `meta-data` because it's unnecessary. The QR code scanner comes in two forms:

- Bundled scanner
- Google Play Store scanner

We use the former and [only the latter requires this](https://developers.google.com/ml-kit/vision/barcode-scanning/android#before-you-begin).